services/time/src/lib.rs

```diff
@@ -115,22 +116,22 @@ impl Transaction for TxTime {
         self.verify_signature(self.pub_key())
     }
 
-    fn execute(&self, view: &mut Fork) {
+    fn execute(&self, view: &mut Fork) -> ExecutionResult {
         let validator_keys = Schema::new(&view).actual_configuration().validator_keys;
 
         // The transaction must be signed by the validator.
         let signed = validator_keys.iter().any(|&validator| {
             validator.service_key == *self.pub_key()
         });
         if !signed {
-            return;
+            return Ok(());
```

Comment:
This calls for an erroneous return ("signed by an invalid author"). Could be dealt with later.

Reply:
There already is an issue for that: https://jira.bf.local/browse/ECR-682
:smile: